RocketChat/Rocket.Chat
# Chore: Engagement Dashboard end to end tests

## Proposed changes (including videos or screenshots)


Adding tests to check the behavior of the Engagement Dashboard for the Enterprise Edition license. 
The tests include: 
- Visibility and navigation of page and tabs
- Fallback component on widgets error 



## Issue(s)


## Steps to test or reproduce

To run tests check the Wiki page: https://github.com/RocketChat/Rocket.Chat/wiki/E2E-Testing

## Further comments
